src/include/daos_pool.h

```diff
@@ -100,6 +110,22 @@ struct daos_pool_space {
 	uint32_t		ps_padding;
 };
 
+/** Pool space usage information v6 */
+struct daos_pool_space_v6 {
```

Comment:
same. this can be hidden in daos/pool.h ?

Reply:
yes, let me see if it could be moved to somewhere else.